index.php

```diff
@@ -82,6 +82,7 @@
 		<title>URL Shortner</title>
         <link rel="stylesheet" href="css/main.css">
         <link rel="stylesheet" href="css/font-awesome-4.6.3/css/font-awesome.min.css">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
Ok you have added meta tag in this PR, It would be great if you can make a single PR with the whole set of changes.